src/pages/Processes.tsx

```diff
@@ -57,12 +58,17 @@ class Processes extends React.PureComponent<IProps, IState> {
             confirmationDialogQuestion: "",
             showExplanation: false,
             showTables: true,
+            runningProcesses: []
         };
     }
 
     updateRunningProcesses = (runningProcesses: ProcessV2[]) => {
-        this.setState({ showTables: false });
-        this.setState({ showTables: true });
+        if (runningProcesses !== this.state.runningProcesses) {
+            this.setState({ showTables: false, runningProcesses });
+            setTimeout(() => {
+                this.setState({ showTables: true });
+            }, 1);
```

Comment:
What does this extra millisecond do?

Reply:
to force rerender of the tables, which doesn't happen without the timeout